src/Probabilities.cpp

```diff
@@ -192,9 +192,7 @@ GroupProbElem::~GroupProbElem()
 {
 	std::map<ProbName, SingleProbElem*>::iterator i;
 	for (i = probs_.begin(); i != probs_.end(); ++i) {
-		SingleProbElem *elem = (*i).second;
-		assert(elem);
```

Comment:
please refrain from removing assertions

Reply:
Why would you insist to keep such a questionable null pointer check (before the usage of a delete expression)?

Reply:
if the application has an invariant, and if a developer felt the need to check this invariant, then a refactoring change should not remove the check. the delete has nothing to do with it.

Reply:
sorry, I don't have time for this